src/frontend/src/styles/components/loading-spinner.scss

```diff
@@ -0,0 +1,3 @@
+.loading-spinner {
+    display: flex;
+}
```

Comment:
I don't think this is needed, if we drop this file it will stay centred. The element already has styling to centre and pad it.

```css
element.style {
    /* display: flex; */
    margin: auto;
    padding: 15%;
}
``